packages/fx-core/templates/plugins/resource/aad/auth/bot/csharp/SSO/TeamsSsoBot.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Teams;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Builder.Dialogs;
+
+namespace {Your_NameSpace}.SSO;
+
+public class TeamsSsoBot<T> : TeamsActivityHandler where T : Dialog
+{
+    private readonly ILogger<TeamsSsoBot<T>> _logger;
+    private readonly BotState _conversationState;
+    private readonly Dialog _dialog;
+    public TeamsSsoBot(ConversationState conversationState, T dialog, ILogger<TeamsSsoBot<T>> logger)
+    {
+        _conversationState = conversationState;
+        _dialog = dialog;
+        _logger = logger;
+    }
+
+    protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
+    {
+        _logger.LogInformation("Receive message activity");
+        var text = turnContext.Activity.Text;
+        var removedMentionText = turnContext.Activity.RemoveRecipientMention();
+        if (!string.IsNullOrEmpty(removedMentionText))
```

Comment:
Did you mean `if (!string.IsNullOrEmpty(text))`

Reply:
Updated, thx!